pandas/core/arrays/categorical.py

```diff
@@ -2071,7 +2071,7 @@ def _reduce(self, name, axis=0, **kwargs):
         return func(**kwargs)
 
     @deprecate_kwarg(old_arg_name="numeric_only", new_arg_name="skipna")
-    def min(self, skipna=True):
+    def min(self, skipna=True, **kwargs):
```

Comment:
IMO adding kwargs just lets users pass wrong arguments and let things fail silently. I'd prefer not passing kwargs.

Reply:
added a TODO to capture this suggestion. this PR is a regression fix to be backported. Only minimal changes to restore previous behaviour for now, avoiding code changes/enhancements.

Reply:
Ok, then I think a good idea would be to check the kwargs using ``pandas.compat.function.validate_min``. This checks that ``{"axis": None}`` is accepted, but ``{"axis": 1}`` fails.

Reply:
yes i agree, we check these elsewhere when we pass kwargs

Reply:
separate PR or here? if here backport the additional code or keep the signature the same as prior to the change that caused the regression?

Reply:
I think it good to do it here: The kwargs are introduced in thus PR, so is reasonable to validate it in the same PR

Reply:
> The kwargs are introduced in this PR

not entirely true. The removal of kwargs caused a regression. the removal is reverted here. the working code did not validate the kwargs.

> I think it good to do it here

will do. can decide what to backport at a later date.